components/mod-event/useModEventList.tsx

```diff
@@ -241,143 +392,50 @@ export const useModEventList = (
     }
   }, [props.eventType])
 
-  const results = useInfiniteQuery<{
-    events: ModEventViewWithDetails[]
-    cursor?: string
-  }>({
-    queryKey: ['modEventList', { listState }],
-    queryFn: async ({ pageParam }) => {
-      const {
-        types,
-        includeAllUserRecords,
-        commentFilter,
-        createdBy,
-        subject,
-        oldestFirst,
-        createdBefore,
-        createdAfter,
-        addedLabels,
-        removedLabels,
-        addedTags,
-        removedTags,
-        reportTypes,
-        policies,
-        limit,
-      } = listState
-      const queryParams: ToolsOzoneModerationQueryEvents.QueryParams = {
-        limit,
-        cursor: pageParam,
-        includeAllUserRecords,
-      }
-
-      if (subject?.trim()) {
-        queryParams.subject = subject.trim()
-      }
-
-      if (createdBy?.trim()) {
-        queryParams.createdBy = createdBy
-      }
-
-      if (createdAfter) {
-        queryParams.createdAfter = new Date(createdAfter).toISOString()
-      }
-
-      if (createdBefore) {
-        queryParams.createdBefore = new Date(createdBefore).toISOString()
-      }
-
-      if (reportTypes.length) {
-        queryParams.reportTypes = reportTypes
-      }
-
-      if (addedLabels.length) {
-        queryParams.addedLabels = addedLabels
-      }
-
-      if (removedLabels.length) {
-        queryParams.removedLabels = removedLabels
-      }
-
-      if (oldestFirst) {
-        queryParams.sortDirection = 'asc'
-      }
+  useEffect(() => {
+    if (!showWorkspaceConfirmation) {
+      abortController.current?.abort('user-cancelled')
```

Comment:
Only `Error` instances should ever be used as abort reason. This is otherwise similar to `throw "some string"`. Note that you can do `ac.abort()` (no argument).